lib/runner.js

```diff
@@ -227,7 +227,7 @@ Runner.prototype.fail = function(test, err) {
   }
 
   ++this.failures;
-  test.state = 'failed';
+  test.state = constants.RUNNER_STATE_FAILED;
```

Comment:
As these two are unused outside "lib/runner.js" itself, why not just let them be module variables?

Reply:
custom reporters which use the `failed` event.

Reply:
Move to `Runner.constants.states`.

Reply:
Most custom reporters probably wouldn't use this, but some may want to listen for e.g., `EVENT_TEST_END` and check `test.state === TEST_STATE_FAILED`, like `xunit`.